content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -278,6 +278,36 @@ kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/
 kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/canal.yaml
 ```
 
+{{% /tab %}}
+
+{{% tab name="Cilium" %}}
+Cilium is open source software for providing and transparently securing network connectivity and loadbalancing between application workloads.
```

Comment:
Most other networking solution tab content doesn't bother with this kind of intro. Can we remove, please? It gets in the way of the useful technical information and provides poor UX on the page. Thanks!